client/src/components/Header/index.js

```diff
@@ -33,6 +33,12 @@ class Header extends Component {
           </li>
         ))
       }
+      <li>
+        <Link to={HELP}>
```

Comment:
I was thinking to add it as another element of the list, but then decided that it is quite special so it was better to put it as special. If in the future there will be more elements like this, I will implement special logic for them.